src/Build/BackEnd/Shared/BuildResult.cs

```diff
@@ -70,6 +71,8 @@ public class BuildResult : INodePacket, IBuildResults
         /// </summary>
         private List<string> _defaultTargets;
 
+        private IReadOnlyCollection<string> _exportTargets;
```

Comment:
Ideally initial and default targets would also become read only collections. Not necessarily in scope for this PR though.

Reply:
ProjectInstance exposes the target lists, so maybe the intention there was to be able to mutate them. But I don't see any good coming from mutating ExportTargets.

Reply:
Good point, it's part of the public interface =/